LEAF_Request_Portal/js/formGrid.js

```diff
@@ -716,9 +721,10 @@ var LeafFormGrid = function(containerID, options) {
 
                 var trimmedText = val.innerText.trim();
                 line[i] = trimmedText;
-                //wrap text that could be interpretted as dates by excel in quotes
+                //add \t to data that could be interpretted as dates by excel
+                const dataFormat = val.getAttribute('data-format');
                 const testDateFormat = /^\d+[\/-]\d+([\/-]\d+)?$/;
-                line[i] = testDateFormat.test(line[i]) ? line[i] + '\t' : line[i];
+                line[i] = dataFormat !== 'date' && testDateFormat.test(line[i]) ? line[i] + '\t' : line[i];
```

Comment:
Would change this back to `"'" + line[i] + "'": line[i];` and get rid of the \t tab that is being metioned